activation/post.go

```diff
@@ -414,10 +414,3 @@ func (mgr *PostSetupManager) LastOpts() *PostSetupOpts {
 func (mgr *PostSetupManager) Config() PostConfig {
 	return mgr.cfg
 }
-
-func (mgr *PostSetupManager) getState() postSetupState {
```

Comment:
rather than removing this method, we could use it where `mgr.state` is read from, i.e. at the beginning of `StartSession` in this file and a few other places

Reply:
👍